app/models/manageiq/providers/autosde/storage_manager/physical_storage.rb

```diff
@@ -10,6 +13,17 @@ def raw_delete_physical_storage
     EmsRefresh.queue_refresh(ems)
   end
 
+  def raw_update_physical_storage(_options = {})
+    update_details = ext_management_system.autosde_client.StorageSystemUpdate(
+      :name           => _options['name'],
+      :password       => _options['password'] || "",
+      :user           => _options['user'] || "",
+      :management_ip  => _options['management_ip'] || "",
+    )
+    ext_management_system.autosde_client.StorageSystemApi.storage_systems_pk_put(ems_ref, update_details)
+    EmsRefresh.queue_refresh(ems)
```

Comment:
(Follow-up) We should get targeted refresh working here so you don't have to queue a full for every one of these: https://www.manageiq.org/docs/guides/providers/targeted_refresh

Reply:
@agrare do you meant we need to do refactoring to the entire autosde provider to use targeted refresh? if so I will open a new ticket for this action, and I prefer to push this PR with the full refresh if its ok with you.

Reply:
> and I prefer to push this PR with the full refresh if its ok with you.

Correct that's what I meant by follow-up

I wrote up a guide on how to do this in https://www.manageiq.org/docs/guides/providers/targeted_refresh

Reply:
I opened an issue for myself to add targeted_refresh for autosde provider. will be done in a separate PR in the short term